Documentation/boards/raspberrypi.md

```diff
@@ -31,8 +31,13 @@ i2c-bcm2708
 
 ## USB Boot
 
-USB mass storage boot is available on Raspberry Pi 3B, 3B+, 3A+, and 2B v1.2.
-To enable USB boot, add `program_usb_boot_mode=1` into `config.txt`. Note that this **permanently** alters the one-time programmable memory of the device.
+USB mass storage boot is available on Raspberry Pi 4 (64-bit only), 3B, 3B+, 3A+, and 2B v1.2.
+
+For Raspberry 3B, 3A+ and 2B v1.2, to enable USB boot, add `program_usb_boot_mode=1` into `config.txt`. Note that this **permanently** alters the one-time programmable memory of the device.
+
+For Raspberry 4, make sure to update the bootloader to a stable release
```

Comment:
@agners I am not sure if this is the right place to bring this up, but do you have any suggestions for people who have previously moved the data partition to an SSD? https://github.com/home-assistant/operating-system/blob/dev/Documentation/partition.md#using-datactl-to-move-the-data-partition

I fall into this category and would love to try the new 5.0 pre-release build, but I don't want to have to start with a clean install on the SSD to do so.